src/main/groovy/life/qbic/oncostore/controller/VariantController.groovy

```diff
@@ -141,12 +141,12 @@ class VariantController {
                             uuid = statusId
                             fileName = file.filename
                             fileSize = file.size
-                            status = Status.processing
+                            status = life.qbic.oncostore.model.Status.processing
```

Comment:
why the prefix? Is there another dependency that introduces a `Status`?

Reply:
There is also a class `Status` in here: `import io.micronaut.http.annotation.*`